doc/nrf/glossary.rst

```diff
@@ -0,0 +1,447 @@
+
+OSS repository
+   An |NCS| repository that tracks an upstream Open Source Software counterpart that is externally maintained.
+
+Over-the-Air (OTA)
+   Any type of wireless transmission.
+
+Patch
+   A method to export git commits for sharing them with others.
+   It does not require a repository.
+
+Power Amplifier (PA)
+   A device used to increase the transmit power level of a radio signal.
+
+Power Saving Mode (PSM)
+   A feature introduced in 3GPP Release 12 to improve battery life of IoT (Internet of Things) devices by minimizing energy consumption.
+   The device stays dormant during the PSM window.
+
+Printed Circuit Board (PCB)
+   A board that connects electronic components.
+
+Protocol Data Unit (PDU)
+   Information transferred as a single unit between peer entities of a computer network and containing control and address information or data.
+   PDU mode is one of the two ways of sending and receiving SMS messages.
+
+Provisioning
+   * In a thread mesh network, the process of associating a device to the appropriate service, and performing any application or vendor-specific configuration.
+     It is a step in the commissioning process.
+   * In a bluetooth mesh network, the process of adding devices to the network.
+   * In a bootloader, the process of storing public key hashes in a separate region of memory from the bootloader image.
+
+Pull Request
+   A set of commits that are sent to contribute to a repository.
+
+Quality of Service (QoS)
+   The measured overall performance of a service, such as a telephony or a computer network, or a cloud computing service.
+
+Radio Co-Processor (RCP)
+   A co-processor offloading radio functions from the host processor.
+   In the |NCS| context, it is typically used in OpenThread and Zigbee platform designs.
+
+Real-time operating system (RTOS)
+   An operating system that reacts to input within a specific period of time.
+   A real-time deadline can be so small that system reaction appears instantaneous.
+
+Real-Time Transfer (RTT)
+   Proprietary technology for bidirectional communication that supports J-Link devices and Arm-based microcontrollers, developed by SEGGER Microcontroller.
+
+Receive Data (RXD)
+   A signal line in a serial interface that receives data from another device.
+
+Repository
+   The entire set of files and folders of which a project is composed, together with the revision history of each file.
+   It is often composed of multiple branches.
+   It is also known as *Git repository* or *Git project*.
+
+Root of Trust (RoT)
+   The property or component in a secure system that provides the foundation of a Chain of Trust.
+
+Sample
+   An implementation example that showcases a single feature or library.
+
+Secure Access Port Protection (SECUREAPPROTECT)
+   A register used to prevent read and write access to all secure CPU registers and secure memory-mapped addresses.
+   See Access Port Protection (APPROTECT).
+
+Security Manager Protocol (SMP)
```

Comment:
There is also the Simple Management Protocol, which unfortunately shares the acronym. Should we explain that as well here to avoid confusion?

Reply:
fixed